examples/client/Locomotion/src/pages/ActiveRide/RideDrawer/RideOptions/ServiceOptions/ServiceCard/index.js

```diff
@@ -0,0 +1,95 @@
+import moment from 'moment';
+import React, { useContext } from 'react';
+import SvgIcon from '../../../../../../Components/SvgIcon';
+import i18n from '../../../../../../I18n';
+import Seat from '../../../../../../assets/seat.svg';
+import { TAG_OPTIONS } from '../../../../../../context/newRideContext/services';
+import { Context as ThemeContext } from '../../../../../../context/theme';
+import {
+  Circle, AvailableSeats,
+  Capacity, CardContainer,
+  CarIcon, Eta,
+  Row, Price,
+  ServiceDetails, TimeDetails,
+  Title, Description,
+  TopRow, CarContainer,
+} from './styled';
+import Tag from '../../../../../../Components/Tag';
+
+const ServiceCard = ({ selected, service }) => {
+  const theme = useContext(ThemeContext);
+  const unavailable = !service.eta;
+  const timeUntilArrival = i18n.t('rideDetails.timeUntilArrival', { minutes: moment.duration(moment(service.eta).diff(moment())).minutes().toString() });
+  const unavailableText = i18n.t('rideDetails.unavailable');
+
+  const tagStyles = {
+    [TAG_OPTIONS.FASTEST]: {
+      container: {
+        backgroundColor: theme.primaryColor,
+      },
+      textColor: theme.primaryButtonTextColor,
+    },
+    [TAG_OPTIONS.CHEAPEST]: {
+      container: {
+        borderWidth: 1,
+        borderStyle: 'solid',
+        borderColor: theme.primaryColor,
+      },
+      textColor: theme.primaryColor,
+    },
+  };
+  return (
+    <CardContainer theme={theme} selected={selected} noBg>
```

Comment:
What is noBg?

Reply:
the button component has a default blue background, so to remove it you add the flag. probably not the best way to do that

Reply:
ok, but there are a bunch of places that use it